yeat/tests/__init__.py

```diff
@@ -54,35 +54,6 @@ def run_yeat(arglist):
     main(args)
 
 
-def get_expected(algorithm, wd, config):
-    analysis_dir = Path(wd).resolve() / "analysis"
-    data = json.load(open(config))
-    expected = []
-    for assembly_label, assembly_obj in data["assemblies"].items():
-        for sample_label in assembly_obj["samples"]:
-            sample_obj = data["samples"][sample_label]
-            short_readtype = set(sample_obj).intersection(ILLUMINA_READS)
-            long_readtype = set(sample_obj).intersection(LONG_READS)
-            if assembly_obj["mode"] in ["paired", "single"]:
-                expected.append(
-                    analysis_dir
-                    / sample_label
-                    / next(iter(short_readtype))
-                    / assembly_label
-                    / assembly_obj["algorithm"]
-                    / FINAL_FILES[algorithm].replace("*", sample_label)
-                )
-            elif assembly_obj["mode"] in ["pacbio", "oxford"]:
-                expected.append(
-                    analysis_dir
-                    / sample_label
-                    / next(iter(long_readtype))
-                    / assembly_label
-                    / assembly_obj["algorithm"]
-                    / FINAL_FILES[algorithm].replace("*", sample_label)
-                )
-    return expected
```

Comment:
This old implementation of calculating the expected final files for each combinatorial combination for samples and assemblies is not needed due to how we calculate them in the objects themselves now.

Reply:
Awesome. I *love* deleting large blocks of code!